lib/src/source/hosted.dart

```diff
@@ -95,7 +95,7 @@ class HostedSource extends Source {
   bool descriptionsEqual(description1, description2) =>
       _parseDescription(description1) == _parseDescription(description2);
 
-  int hashDescription(description) => _parseDescription.hashCode;
+  int hashDescription(description) => description.hashCode;
```

Comment:
This probably has to be `_parseDescription(description).hashCode`, then I think we're on right track.

I wonder if this will yield any performance improvements :)